pkg/gui/controllers/commit_description_controller.go

```diff
@@ -75,6 +75,22 @@ func (self *CommitDescriptionController) switchToCommitMessage() error {
 	return nil
 }
 
+func (self *CommitDescriptionController) handleTogglePanel() error {
```

Comment:
Something that is not clear here is that `opts.Config.Universal.TogglePanel` defaults to `tab`. So a reader could get confused as to why we're conditionally inserting spaces. A comment should clear up that confusion.

Things probably get awkward when a user remaps these keys, and I'm not sure of a good way of dealing with that (doesn't need to be solved in this PR)

Reply:
Ah interesting, I hadn't considered remapping keys at all. I added a condition and a comment in 8256ed219e7a, please have a look.